eventkit_cloud/utils/provider_check.py

```diff
@@ -430,14 +430,24 @@ def get_bbox(self, element):
         return bbox
 
 
+class TMSProviderCheck(ProviderCheck):
+
+    def __init__(self, service_url, layer, aoi_geojson=None, slug=None):
+        super(TMSProviderCheck, self).__init__(service_url, layer, aoi_geojson, slug)
+        if not self.service_url.endswith('/'):
+            self.service_url += '/'
+        self.service_url += '0/0/0.png'
```

Comment:
It could be more concise to just .rstrip('/') endings and manually add them in.... 
`self.service_url = "{0}/0/0/0.png".format(self.service_url.rstrip('/'))`